de/cheatsheet-deep-learning.md

```diff
@@ -60,7 +60,7 @@
 
 **11. Learning rate ― The learning rate, often noted α or sometimes η, indicates at which pace the weights get updated. This can be fixed or adaptively changed. The current most popular method is called Adam, which is a method that adapts the learning rate.**
 
-&#10230; Lernrate - Die Lernrate, oft mit α oder manchmal mit η bezeichnet, gibt an mit welcher Schnelligkeit die Gewichtungen aktualisiert werden. Die Lernrate kann konstant oder anpassend variierend sein. Die aktuell populärste Methode, Adam, ist eine Methode die die Lernrate anpasst.
+&#10230; Lernrate - Die Lernrate, oft mit α oder manchmal mit η bezeichnet, gibt an mit welcher Rate die Gewichtungen aktualisiert werden. Die Lernrate kann konstant oder anpassend variierend sein. Die aktuell populärste Methode, Adam, ist eine Methode die die Lernrate anpasst.
```

Comment:
~~Gewichtungen~~ but:  Gewichte

*) Die Lernrate kann konstant oder ~~anpassend variierend sein.~~ **dynamisch angepasst werden.** 
*) ~~Die aktuell populärste Methode, Adam, ist eine Methode die die Lernrate anpasst.~~ Am häufigsten wird die Methode Adam benutzt, welche die Lernrate dynamisch aktualisiert.

So final correct one: 
Lernrate - Die Lernrate, oft mit α oder manchmal mit η bezeichnet, gibt an mit welcher Rate die _Gewichte_ aktualisiert werden. Die Lernrate kann konstant oder _dynamisch angepasst werden._ _Am häufigsten wird die Methode Adam benutzt, welche die Lernrate dynamisch aktualisiert._

Reply:
Very good, thank you! Perhaps also "die Adam-Methode" insetad of "die Methode Adam"?

Reply:
Adam-Methods sounds nicer, indeed

Reply:
Now "Die aktuell populärste Methode, Adam, aktualisiert die Lernrate dynamisch." Good?